### ISSUE TITLE
A Basic YouTube Clone using HTML and CSS for beginners

### ISSUE THREAD

**Issue by harshit-sharma-gits** [codedamn/projects#82]
Hi! I would like to add a YouTube Clone here if you would allow. The clone can be made using HTML and CSS only so that beginners can also understand the codebase and attempt to give it a try by making it yourself.
May I go ahead with this?
If yes, please assign this to me with `hacktoberfest` label.

**Comment by Pranavtechie** [comment]
Hey @harshit-sharma-gits , you can go ahead with making the guided-project version of the YouTube clone project. 

Please make sure that you understand the mentioned instructions, before starting

**Comment by harshit-sharma-gits** [comment]
Hey @Pranavtechie, I have opened up a Pull Request (#113 ) for this issue, would you please take a look?

**Comment by aadii0408** [comment]
Hey, I want to work on this issue, please assign to me.

**Comment by harshit-sharma-gits** [comment]
I believe this issue can be closed now, as the guided project have been successfully merged into main.
What you say @Pranavtechie ?

And, is there anything more to do for receiving codedamn swags? XD

**Comment by Pranavtechie** [comment]
Hey @harshit-sharma-gits , drop your email here. We'll reach out to you

**Comment by harshit-sharma-gits** [comment]
> Hey @harshit-sharma-gits , drop your email here. We'll reach out to you

Here;
harshits908@gmail.com
